### ISSUE TITLE
Avoid manually referencing line numbers in error messages

### ISSUE THREAD

**Issue by AlejoAsd** [marcoshuck/go-distributed#3]
https://github.com/marcoshuck/go-distributed/blob/c3d8745480eeff95a7cb6885480983de55b3cb87/challenge/pkg/server/server.go#L39

This panic message is attempting to include the line where `panic` was called. However, it seems the code has changed since this line was written, and now the error message and the line don't match.

In general, it is not a good idea to write down line numbers in strings, as strings are quite static and line number very dynamic. The stack trace should provide this information.

**Comment by marcoshuck** [comment]
I agree, I didn't invest enough time to understand how aws-sdk-go does this logging filename + line format. I'll work on that.
